base.c

```diff
@@ -79,6 +79,15 @@ char* str_tokens[] = {
 
 int tokens_size = sizeof str_tokens / sizeof *str_tokens;
 
+int findAddress(char *value){
```

Comment:
Could you please rename this method to "findVariableAddress" or something like that, because we're also going to need a "findLabelAddress" function later